src/member/forms.py

```diff
@@ -250,7 +250,7 @@ class MemberForm(forms.Form):
     member = forms.CharField(
         label=_("Member"),
         widget=forms.TextInput(attrs={
-            'placeholder': _('Member'),
+            'placeholder': _('Type more than 2 characters to search Members'),
```

Comment:
Maybe a rewording as "Type 3 characters or more to search members"? I feel it's more clear that the user has to put at least 3 characters.